streams/src/main/java/org/apache/kafka/streams/kstream/UnlimitedWindows.java

```diff
@@ -38,12 +39,18 @@ private UnlimitedWindows(String name, long start) {
     }
 
     /**
-     * Returns an unlimited window definition
+     * Return an unlimited window definition TODO ?
```

Comment:
This sounds confusing (both the original and the updated docstring) -- "return an unlimited window definition"?